game-core/src/main/java/games/strategy/debug/ClientLogger.java

```diff
@@ -1,100 +0,0 @@
-package games.strategy.debug;
-
-import static com.google.common.base.Preconditions.checkNotNull;
-
-import java.awt.GraphicsEnvironment;
-import java.io.PrintStream;
-import java.util.concurrent.ExecutionException;
-
-import javax.annotation.Nullable;
-
-import com.google.common.annotations.VisibleForTesting;
-
-/**
- * Provides methods for the client to write log messages.
- * <p>
- * In general, the {@code logError()} methods will send their output to the user output
- * (standard error) stream, while the {@code logQuietly()} methods will send their output to the
- * developer output (standard output) stream.
- * </p>
- */
-public final class ClientLogger {
```

Comment:
I migrated existing calls incrementally by first having ClientLogger use a logger itself. I then checked things, launched a game, demo'ed error messages to verify that I had successfully routed logger messages to the console (so the very first thing I did was to remove the logging.properties file and wire messages to the console logger directly).

So this was then a confidence building step here with ClientLogger using just a logger. The next step I took after that was to convert all System.out's and System.err'rs, then to finish it off I replaced ClientLogger calls with logger calls. Having ClientLogger call logger itself really gave confidence that things like the following would be redundant:

```
ClientLogger.log...
log.warning("...")
```